types/web-bluetooth/index.d.ts

```diff
@@ -169,6 +169,17 @@ interface Bluetooth extends EventTarget, BluetoothDeviceEventHandlers, Character
     addEventListener(type: string, listener: EventListenerOrEventListenerObject, useCapture?: boolean): void;
 }
 
+interface BluetoothUUID {
+    getService(name: string | number): string;
+    getCharacteristic(name: string | number): string;
+    getDescriptor(name: string | number): string;
+    canonicalUUID(alias: string | number): string;
+}
+
 interface Navigator {
     bluetooth: Bluetooth;
 }
+
+interface Window {
```

Comment:
Is it normal to specify the window interface? `BluetoothUUID` may appear on other global objects, too.

Reply:
I'm not sure what the standard/correct practice is. (I'm relatively new to TS) According to the spec BluetoothUUID is exposed on `Window` only:

```java
[Exposed=Window]
interface BluetoothUUID {
  static UUID getService((DOMString or unsigned long) name);
  static UUID getCharacteristic((DOMString or unsigned long) name);
  static UUID getDescriptor((DOMString or unsigned long) name);
  static UUID canonicalUUID([EnforceRange] unsigned long alias);
};
```

but if you want I can declare it as a general global variable so you can access it without doing `window.BluetoothUUID`:
```typescript
declare const BluetoothUUID: BluetoothUUID;
```

Reply:
Looking at the existing `Navigator` type, you may just be able to omit the `Window` declaration entirely?

Keen from input from others here, though.

Reply:
If I omit the whole `Window` declaration, then `BluetoothUUID` won't be exposed. I can do the `declare const` instead

Reply:
I looked at some other libraries and it _seems_ like the standard practice would be something like:
```typescript
declare namespace BluetoothUUID {
    // Functions go here
};
```

Reply:
Interesting, I wonder how the navigator interface in this type gets picked up?

Reply:
I _think_ (I could be wrong) it's because all the interfaces and types declared in index.d.ts are in their own scope, but `declare`-ing variables/namespaces allows them to be picked up.

In addition, adding new properties to existing interfaces that external files know about, like `Window` and `Navigator`, allows them to be picked up.

I tried using `declare namespace` and it works perfectly! I'm able to access `BluetoothUUID` with `BluetoothUUID`, `window.BluetoothUUID`, and `globalThis.BluetoothUUID`. I'll go ahead and change this to use `declare namespace`